File: MMS-API/DTOs/SCarManageRecordDto.cs
using System;
using API.Helpers;

namespace MMS_API.DTOs
{
    public class SCarManageRecordDto : PaginationParams
    {
        public string LicenseNumber { get; set; }
        public string SignInDateS { get; set; }
        public string SignInDateE { get; set; }
    }
}

File: MMS-SPA/src/app/core/_models/s-car-manage-record-dto.ts
import { Pagination } from "./pagination";

export class SCarManageRecordDto extends Pagination {
  licenseNumber: string;
  signInDateS: string;
  signInDateE: string;

  constructor() {
    super();
    this.licenseNumber = "";
    this.signInDateS = "";
    this.signInDateE = "";
    this.isPaging = true; //開分頁
  }
  public setPagination(pagination: Pagination) {
    this.currentPage = pagination.currentPage;
    this.itemsPerPage = pagination.itemsPerPage;
    this.totalItems = pagination.totalItems;
    this.totalPages = pagination.totalPages;
  }
}
